ui/src/app/pages/admin/user/admin-user-bypass.spec.tsx

```diff
@@ -1,40 +1,100 @@
 import * as React from 'react';
 import { mount } from 'enzyme';
 
-import { AdminTableUser } from 'generated/fetch';
-
+import { Button } from 'app/components/buttons';
 import { serverConfigStore } from 'app/utils/stores';
 
+import defaultServerConfig from 'testing/default-server-config';
 import { ProfileStubVariables } from 'testing/stubs/profile-api-stub';
 
 import { AdminUserBypass } from './admin-user-bypass';
 
 describe('AdminUserBypassSpec', () => {
-  let props: { user: AdminTableUser };
+  const defaultProps = {
+    user: ProfileStubVariables.ADMIN_TABLE_USER_STUB,
+  };
 
   const component = () => {
-    return mount(<AdminUserBypass {...props} />);
+    return mount(<AdminUserBypass {...defaultProps} />);
   };
 
   beforeEach(() => {
+    serverConfigStore.set({
+      config: defaultServerConfig,
+    });
+  });
+
+  it('should render bypass toggles for access modules', () => {
     serverConfigStore.set({
       config: {
-        gsuiteDomain: 'fake-research-aou.org',
-        projectId: 'aaa',
-        publicApiKeyForErrorReports: 'aaa',
+        ...serverConfigStore.get().config,
         enableEraCommons: true,
         enableRasLoginGovLinking: true,
+        enableComplianceTraining: true,
       },
     });
-    props = {
-      user: ProfileStubVariables.ADMIN_TABLE_USER_STUB,
-    };
+
+    const wrapper = component();
+    wrapper.find(Button).simulate('click');
+
+    [
+      'rt-compliance-training-toggle',
+      'ct-compliance-training-toggle',
+      'ducc-toggle',
+      'era-commons-toggle',
+      'two-factor-auth-toggle',
+      'ras-link-login-gov-toggle',
+      'profile-confirmation-toggle',
+      'publication-confirmation-toggle',
+    ].forEach((dataTestId) => {
+      if (!wrapper.find(`[data-test-id="${dataTestId}"]`).exists()) {
+        console.log('dataTestId: ' + dataTestId);
+      }
+      expect(
+        wrapper.find(`[data-test-id="${dataTestId}"]`).exists()
+      ).toBeTruthy();
```

Comment:
I love that you are providing a more helpful failure message here than what would be given without the console.log. As a matter of habit, I would avoid two `wrapper.find` calls. Since expect has a limited API, perhaps something like:
```suggestion
    expect([
      'rt-compliance-training-toggle',
      'ct-compliance-training-toggle',
      'ducc-toggle',
      'era-commons-toggle',
      'two-factor-auth-toggle',
      'ras-link-login-gov-toggle',
      'profile-confirmation-toggle',
      'publication-confirmation-toggle',
    ].filter(id => !wrapper.find(`[data-test-id="${id}"]`).exists())
    .map(id => console.log('dataTestId: ' + id))
    .length).toBe(0) // count of not found data-test-ids
```

Reply:
Good idea, done.

Ideally, we'd have a library that bridges jest and enzyme to allow assertions like `expect(wrapper).toContain(...)` with good error messages.

Reply:
Indeed. Or try out https://volument.com/baretest